src/main/cli/settings.c

```diff
@@ -1274,6 +1275,11 @@ const clivalue_t valueTable[] = {
     { "vtx_halfduplex",             VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_VTX_CONFIG, offsetof(vtxConfig_t, halfDuplex) },
 #endif
 
+// PG_VTX_IO
+#ifdef USE_VTX_RTC6705
+    { "flash_spi_bus",              VAR_UINT8 | MASTER_VALUE, .config.minmaxUnsigned = { 0, SPIDEV_COUNT }, PG_VTX_IO_CONFIG, offsetof(vtxIOConfig_t, spiDevice) },
```

Comment:
`flash`?

Also, please make this a `HARDWARE_VALUE` to get it included in `dump hardware`.

Reply:
sure, let me fix that now.